### DISCUSSION TITLE
coc diagnostics background color

### DISCUSSION THREAD

**Discussion by ElcomJ** [neoclide/coc.nvim#3979] [Q&A] [answered]
Good evening, I have tried a few ways but unfortunately I can't.

I would like to know if it is possible to change the yellow background color of the coc diagnostic window to another color, and if so which highlight would I need to change?

Image of the color I would like to change:
![Captura de tela de 2022-08-01 18-08-27](https://user-images.githubusercontent.com/37147897/182247023-9894c0c1-6879-4805-a258-e679a46d70af.png)

Thanks in advance and congratulations on your Plugin, I love it :)

**Comment by chemzqm** [comment]
Try change highlight of CursorLine

**Comment by ElcomJ** [comment]
Good morning, I already have a highlight configuration for the CursorLine, it works when I move between the other errors in the coc diagnostics, but I would like to change the yellow color of the error when it is selected.

**Comment by chemzqm** [comment]
Don't know where it's from, try change highlight for `CocListLine`

**Comment by ElcomJ** [comment]
I tried to modify it, but it had no effect.

Anyway, I will end our conversation right here, this is just a visual modification, nothing too much.

Thanks for your attention and have a great day. :)

**Comment by ElcomJ** [comment]
Just one more thing.

Congratulations on the latest coc.nvim update, I love the new colors of the autocomplete menu, it's perfect.

**Comment by chemzqm** [comment, marked as answer]
Seems it's `Search` highlight used by vim's location list.

**Comment by ElcomJ** [comment]
Hello, recently searching the internet I managed to find a post with the [solution](https://vi.stackexchange.com/questions/28231/disable-highlighting-in-location-list-in-vim) to my problem, and as you said it was a highlight of the `location list,` more specifically of the `QuickFixLine`.

The command below solved my problem:

`highlight QuickFixLine ctermbg=none`
